### PR TITLE
101 keyword inconsistencies

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Often the keywords /
Are inconsistent or wrong /
This mitigates that.

There are still keyword related issues, <https://github.com/18F/omb-eregs/issues/103>, <https://github.com/18F/omb-eregs/issues/104>, and <https://github.com/18F/omb-eregs/issues/105>, but since they require information from OMB I'll address them in separate branches/PRs.

### CODE REVIEW

File: reqs/management/commands/import_reqs.py [hunk 1]
```diff
@@ -21,43 +24,63 @@
...
+KEYWORDS = {
+    "Commodity It": ["Commodity IT"],  # This one is due to str.title()
+    "Data Management/Standards. Reporting": [
+        "Data Management/Standards",
+        "Reporting"
+    ],
+    "Definition": ["Definitions"],
+    "Emergency Preparedness?": ["Emergency Preparedness"],
+    "Record Management": ["Records Management"],
+}
```
Comment (cmc333333) [inline reqs/management/commands/import_reqs.py]: I like the declarative style

File: reqs/management/commands/import_reqs.py [hunk 2]
```diff
@@ -111,12 +134,35 @@ def __init__(self, fields):
         self.cache = {}
         self.fields = fields
 
+    def normalize_keywords(self, values):
```
Comment (cmc333333) [inline reqs/management/commands/import_reqs.py]: Should this be a static method? `self` isn't used.

File: reqs/management/commands/import_reqs.py [hunk 3]
```diff
@@ -111,12 +134,35 @@ def __init__(self, fields):
...
+        """
+        Executive decisions:
+        +   Keywords cannot contain whitespace at the start or end.
+        +   All whitespace will be converted to a single space.
+        +   Keyword values must be init-caps.
+        +   Use of hyphens as separators requires a single space around the
+            hyphen.
```
Comment (cmc333333) [inline reqs/management/commands/import_reqs.py]: I'm uncertain about these last two, but I'm cool with using them to clean the data; I don't think they can hurt.

File: reqs/tests/import_reqs_tests.py [hunk 4]
```diff
@@ -251,3 +258,39 @@ def test_varying_policy_headers(opfid_field, uri_field):
...
+    Test that we don't add keywords with different capitalization into the
+    data.
+    """
+    csv_str = "\n".join([
+        SAMPLE_CSV2[0],
+        SAMPLE_CSV2[1].replace("Software Development Lifecycle/Agile", keyword)
+    ])
+    blank_csv_file.write(csv_str)
+
+    call_command('import_reqs', str(blank_csv_file))
```
Comment (cmc333333) [inline reqs/tests/import_reqs_tests.py]: No biggie, but do you think that we could avoid all of this setup (e.g. by just calling `keywords`)? The more setup, the more complicated the tests are to understand -- for example, I had to jump to the top of the file to figure out why "Software" was in this list.

File: reqs/management/commands/import_reqs.py (hunk 3 above)
Comment (tadhg-ohiggins) [inline reqs/management/commands/import_reqs.py]: I'm not adamant that these should be the standards, but I do think we need standards of some kind for consistency.

File: reqs/tests/import_reqs_tests.py (hunk 4 above)
Comment (tadhg-ohiggins) [inline reqs/tests/import_reqs_tests.py]: Good point on “Software” being somewhat random. I still wanted this to be more comprehensive than just keywords, though. My change (which I'll commit shortly) here is to add a new line to the sample CSV that doesn't include the `x` in the Software row and to test against that instead, removing that bit of mystery.

**Pull request merged**